client/container.go

```diff
@@ -298,30 +298,20 @@ func (c *Client) ContainerList(ctx context.Context, ownerID user.ID, prm PrmCont
 	return &res, nil
 }
 
-// PrmContainerDelete groups parameters of ContainerDelete operation.
-type PrmContainerDelete struct {
+// PrmContainerDeleteOptional groups optional parameters of ContainerDelete operation.
+type PrmContainerDeleteOptional struct {
```

Comment:
I'm not sure about these renames. They kinda OK, since these parameters really are optional and at the same time this makes the type name somewhat long and their 'optional' status is pretty clear at the same time both from the way they're used and type doc.

Reply:
Ok, I will remove `Optional` from names

Reply:
Removed